healpy/src/_sphtools.pyx

```diff
@@ -83,7 +85,18 @@ def map2alm_spin_healpy(maps, spin, lmax = None, mmax = None):
     -------
     alms : list of 2 arrays
       list of 2 alms
+
+    Warnings
```

Comment:
can you please use the `warnings` directive? https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#warnings

Reply:
Since you’re using numpydoc, that’s effectively what I have done. See the docs. Unless you want me to specifically use the directive for some reason?